docs/components/date_pickers.md

```diff
@@ -34,6 +33,10 @@ Date pickers can be embedded into:
 
 Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
 
+### Types
+
+There are two types of date pickers: 1\. [date pickers](#date-pickers) 2\. [date range pickers](#date-range-pickers)
```

Comment:
what is the \ for?

Reply:
I believe thats what makes it render as a list on material.io

See https://material.io/components/tabs/flutter#using-tabs for example